Venmo/src/main/java/com/braintreepayments/api/VenmoRequest.java

```diff
@@ -93,11 +106,147 @@ String getPaymentMethodUsageAsString() {
         }
     }
 
+    /**
+     * @param flag Optional. Whether or not shipping address should be collected and displayed on Venmo paysheet.
+     */
+    public void setCollectCustomerShippingAddress(boolean flag) {
+        this.collectCustomerShippingAddress = flag;
+    }
+
+    /**
+     * @return The boolean value of the flag that signifies whether customer shipping address will be collected.
+     */
+    public boolean getCollectCustomerShippingAddress() {
+        return collectCustomerShippingAddress;
+    }
+
+    String getCollectCustomerShippingAddressAsString() {
+        return String.valueOf(this.collectCustomerShippingAddress);
+    }
+
+    /**
+     * @param flag Optional. Whether or not billing address should be collected and displayed on Venmo paysheet.
+     */
+    public void setCollectCustomerBillingAddress(boolean flag) {
+        this.collectCustomerBillingAddress = flag;
+    }
+
+    /**
+     * @return The boolean value of the flag that signifies whether customer billing address will be collected.
+     */
+    public boolean getCollectCustomerBillingAddress() {
+        return collectCustomerBillingAddress;
+    }
+
+    String getCollectCustomerBillingAddressAsString() {
+        return String.valueOf(this.collectCustomerBillingAddress);
+    }
+
+    /**
+     * @param subTotalAmount Optional. The subtotal amount of the transaction, excluding taxes, discounts, and shipping.
+     */
+    public void setSubTotalAmount(String subTotalAmount) {
```

Comment:
```suggestion
    public void setSubTotalAmount(@Nullable String subTotalAmount) {
```
We need the setters need the same nullability annotations for Kotlin interop.